data/src/test/java/org/oppia/data/backends/test/MockTopicTest.kt

```diff
@@ -0,0 +1,57 @@
+package org.oppia.data.backends.test
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import okhttp3.OkHttpClient
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.data.backends.api.MockTopicService
+import org.oppia.data.backends.gae.NetworkInterceptor
+import org.oppia.data.backends.gae.NetworkSettings
+import org.oppia.data.backends.gae.api.TopicService
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import retrofit2.mock.MockRetrofit
+import retrofit2.mock.NetworkBehavior
+
+/**
+ * Test for [TopicService] retrofit instance using [MockTopicService]
+ */
+@RunWith(AndroidJUnit4::class)
+class MockTopicTest {
+  private lateinit var mockRetrofit: MockRetrofit
+  private lateinit var retrofit: Retrofit
+
+  @Before
+  @Throws(Exception::class)
```

Comment:
Nit here & below: not needed since these are called reflectively by JUnit so the Java runtime annotation isn't necessary.

Reply:
Solved. Removed `@Throws`